src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -129,6 +129,56 @@ public async Task SendPingAsync_InvalidArgs()
             AssertExtensions.Throws<ArgumentException>("buffer", () => { p.Send(TestSettings.LocalHost, 1, new byte[65501]); });
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        public async Task SendPingAsync_InvalidArgs_TimeSpan()
+        {
+            IPAddress localIpAddress = await TestSettings.GetLocalIPAddressAsync();
+            Ping p = new Ping();
+
+            // Null address
+            AssertExtensions.Throws<ArgumentNullException>("address", () => { p.SendPingAsync((IPAddress)null); });
+            AssertExtensions.Throws<ArgumentNullException>("hostNameOrAddress", () => { p.SendPingAsync((string)null); });
+            AssertExtensions.Throws<ArgumentNullException>("address", () => { p.SendAsync((IPAddress)null, null); });
+            AssertExtensions.Throws<ArgumentNullException>("hostNameOrAddress", () => { p.SendAsync((string)null, null); });
+            AssertExtensions.Throws<ArgumentNullException>("address", () => { p.Send((IPAddress)null); });
+            AssertExtensions.Throws<ArgumentNullException>("hostNameOrAddress", () => { p.Send((string)null); });
```

Comment:
how are these duplicated tests relevant to the new method? there's no timespan.

Reply:
Oh. My mistake. I'll change this